src/backend/api/Fusion.Resources.Domain/Commands/Personnel/ReplaceProjectPositionInstancesAssignedPerson.cs

```diff
@@ -0,0 +1,91 @@
+﻿using MediatR;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Fusion.ApiClients.Org;
+
+namespace Fusion.Resources.Domain.Commands
+{
+    public class ReplaceProjectPositionInstancesAssignedPerson : TrackableRequest
+    {
+        public ReplaceProjectPositionInstancesAssignedPerson(Guid projectId, Guid contractIdentifier, PersonnelId fromPerson, PersonnelId toPerson)
+        {
+            OrgProjectId = projectId;
+            OrgContractId = contractIdentifier;
+            FromPerson = fromPerson;
+            ToPerson = toPerson;
+        }
+        public Guid OrgContractId { get; }
+        public Guid OrgProjectId { get; }
+        public PersonnelId FromPerson { get; }
+        public PersonnelId ToPerson { get; }
+
+        public class Handler : AsyncRequestHandler<ReplaceProjectPositionInstancesAssignedPerson>
+        {
+            private readonly IOrgApiClient client;
+
+            public Handler(IOrgApiClientFactory orgApi)
+            {
+
+                this.client = orgApi.CreateClient(ApiClientMode.Application);
+            }
+
+            protected override async Task Handle(ReplaceProjectPositionInstancesAssignedPerson request, CancellationToken cancellationToken)
+            {
+                ApiDraftV2? draft = null;
+                try
+                {
+                    draft = await CreateProvisionDraftAsync(request);
+
+                    var positions =
+                        await client.GetContractPositionsV2Async(request.OrgProjectId, request.OrgContractId);
+
+                    await ReplaceAssignedPersonOnRelevantPositionInstancesInDraftAsync(request, draft, positions);
```

Comment:
What about current position, should that be split?

Reply:
All position instances being assigned to the expired person should be replaced by new account.